### PR TITLE
tinymce version 4 support added

### PR DESCRIPTION
**Pull request by dimaninc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,13 +18,13 @@ Find a "config.js" file an edit it:
 
     var UPLOADCARE_PUBLIC_KEY = "demopublickey";
 
-Add a plugin and plugin button when initializing TinyMCE:
+Add the plugin and plugin button when initializing TinyMCE:
```
Comment (dmitry-mukhin) [inline README.md]: :+1:

File: plugin.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
-UPLOADCARE_CROP = true;
+UPLOADCARE_CROP = '';
 UPLOADCARE_AUTOSTORE = true;
```
Comment (dmitry-mukhin) [inline plugin.js]: this setting is obsolete for widget 2.5.1

**Comment by dmitry-mukhin** [comment]
nice.
- [  ] update readme
- [  ] create branch from current master to support TinyMCE 3 users

**Comment by dmitry-mukhin** [comment]
- [ ] a task list item
- [ ] list syntax required

File: plugin.js (hunk 2 above)
Comment (dimaninc) [inline plugin.js]: which one? or both?

Comment (dmitry-mukhin) [inline plugin.js]: this thread is under `UPLOADCARE_AUTOSTORE`

Comment (dmitry-mukhin) [inline plugin.js]: and `UPLOADCARE_CROP` could be made configurable. see https://github.com/uploadcare/uploadcare-redactor/blob/master/uploadcare.js

**Comment by dimaninc** [comment]
added user crop settings support
updated readme
created a branch with original plugin for tinymce 3 users

**Comment by dmitry-mukhin** [comment]
- [] remove .gitmodules

**Comment by dimaninc** [comment]
done

**Pull request merged**